### PR TITLE
Api gateway - Ready for QA

### PR DESCRIPTION
**Pull request by matthewkeogh50**
initial API GW page

### CODE REVIEW

File: docs/aws/services/API_Gateway.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+### Request Throttling
+Requests to API GW are throttled by default to prevent abuse.
+- 10,000 requests per second (RPS) per gateway.
+- 5,000 request burst limit that applies across all APIs within an account.
+- Returns 429 Too Many Requests error response when exceeded.
+- Can be increased on request (to AWS support or via Service Quotas).
+
+![image](/img/api_gw_throttle.png)
+
+### Catching
```
Comment (NJonesUK) [inline docs/aws/services/API_Gateway.md:37]: Spelling

File: docs/aws/services/API_Gateway.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+### Authentication
+
+- Resource based policies are supported in API GW, take care when configuring these and always enforce conditional checks.
+- Private API's can only be called from within the VPC.
+- API keys are supported for API's hosted in API GW.
+- Amazon Cognito user pools can be used for API auth.
+
+### API GW Authorizers
+
+Authorizers provide the ability to authenticate users/systems with an API endpoint via a Lambda function or Cognito user pool. API GW will create a resource policy based on the chosen function or user pool.
```
Comment (NJonesUK) [inline docs/aws/services/API_Gateway.md:54]: Can we expand a bit on how authorizers work?

File: docs/aws/services/API_Gateway.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+![image](/img/api_gw_domain_name.png)
+
+
+
+### Data Security
+
+- REST API caching can be encrypted.
+- To ensure HTTP requests to your back-end services are originating from API Gateway, you can use Client Certificates to verify the requester's authenticity.
+- TLS and SSL support available.
```
Comment (NJonesUK) [inline docs/aws/services/API_Gateway.md:77]: What do we mean here? SSL's the old version of the TLS protocol, right? Are they still supporting SSLv3 for some reason?

**Review by NJonesUK — CHANGES_REQUESTED** [review]

File: docs/aws/services/API_Gateway.md (hunk 3 above)
Comment (matthewkeogh50) [reply docs/aws/services/API_Gateway.md:77]: my mistake - you're right. SSLv3 is not supported according to https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-custom-domain-tls-version.html 

Needs to be at least TLSv1 - I'll update

**Review by NJonesUK — APPROVED** [review]

**Pull request merged**